.circleci/config.yml

```diff
@@ -54,14 +54,14 @@ commands:
             - save_cache:
                 paths:
                   - ./.tox
-                key: v0.28-toxenv-{{ .Environment.CIRCLE_BRANCH }}-{{ .Environment.CIRCLE_JOB }}-{{ checksum "tox.ini" }}-{{ checksum "setup.py" }}-{{ checksum "requirements.txt" }}-{{ checksum "requirements_dev.txt" }}
+                key: v0.29-toxenv-{{ .Environment.CIRCLE_BRANCH }}-{{ .Environment.CIRCLE_JOB }}-{{ checksum "tox.ini" }}-{{ checksum "setup.py" }}-{{ checksum "requirements.txt" }}-{{ checksum "requirements_dev.txt" }}
```

Comment:
Could you please educate me, when should this be ++'d?

Reply:
Ah good question, I did it now to force it not to use the cache. I don't think we have a good protocol about how we update it. One thing would be obviously if we change requirements, or config changes. I think this is a balance between lowering our cost/time with circleci and hitting case as this one that you discovered.

@raubitsj could probably add more context :)

Reply:
Got it, thanks. Definitely deserves a section in CONTRIBUTING.md :)
Can't wait to get this merged in, it'll make my Settings PR unbelievable green!